app/namespaces/test_adjustment/test_adjustment_service.py

```diff
@@ -111,8 +119,7 @@ def fit_one_pystan_model(self, model_params: Dict) -> Tuple:
                                           iter=self.n_iter,
                                           chains=self.n_chains,
                                           control={'adapt_delta': adapt_delta},
-                                          check_hmc_diagnostics=False,
-                                          init=init)
```

Comment:
why did we scrap this `init`?

Reply:
because it was breaking the initialization of test adj in some cases and we dont know why and rahul said it was fine to leave out